compiler/rustc_passes/src/check_attr.rs

```diff
@@ -544,6 +544,37 @@ impl CheckAttrVisitor<'tcx> {
                         {
                             return false;
                         }
+                    } else if let Some(i_meta) = meta.meta_item() {
+                        if ![
+                            sym::cfg,
+                            sym::hidden,
+                            sym::html_favicon_url,
+                            sym::html_logo_url,
+                            sym::html_playground_url,
+                            sym::html_root_url,
+                            sym::include,
+                            sym::inline,
+                            sym::issue_tracker_base_url,
+                            sym::masked,
+                            sym::no_default_passes, // deprecated
```

Comment:
I just realized this attribute isn't documented; is that intentional? If so, I think we should remove the docs for `--passes` and `--no-default-passes`. Really the whole https://doc.rust-lang.org/nightly/rustdoc/passes.html chapter looks questionable now that customizing passes is deprecated.

Reply:
Let's open an issue and discuss about it then.